docs: add PE-5567 to release notes (#5081)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

  [Build Provider Images](../clusters/edge/edgeforge-workflow/palette-canvos/build-provider-images.md) for guidance on
  using the **certs** folder to pass multiple proxy certificates to the CanvOS build process.

<!-- prettier-ignore-start -->

- The Edge <VersionedLink text="BYOOS" url="/integrations/packs/?pack=edge-native-byoi" /> pack has new parameters that
  allow you to configure node draining behavior during cluster upgrades or repaves. Previously, nodes were always
  drained during upgrades and repaves, even for single-node clusters. Refer to
  [Skip Node Draining](../clusters/edge/cluster-management/skip-draining.md) for guidance on configuring draining
  behavior.

  <!-- prettier-ignore-end -->

> 

Reviewer: ```suggestion
#### Bug Fixes

```

@lennessyy  This feels like it should be under a Bug Fixes section, right?

Author: I debated this because there is no bug fix section for any other product. But I think yeah bug fix does make more sense. I was thinking it was technically already not a bug in 4.5.x. But it did start as a bug, so yes i will move it.

- Upgrading the Palette agent from 4.4.x to 4.5.11 and later will now automatically renew the Certificate Authority (CA)
  certificate for `stylus-webhook` Mutating Webhook Configuration. The corresponding Palette version for Palette agent
  4.5.11 is 4.5.15.

  This was an issue that affected 4.4.x and prior versions and was partially addressed in 4.5.0. The new version fully
  addresses the issue by automatically renewing the CA certificate for 10 years during an upgrade. In previous 4.5.x
  versions, while you would not encounter the certificate expiry issue if your cluster was created using a 4.5.x version

>   of the Palette agent, upgrading from 4.4.x would not have renewed the certificate automatically.

Reviewer: @lennessyy This feels like it should be under the Bug Fixes section. Feel free to add it to the Edge section.

  Previously, nodes were always drained during upgrades and repaves, even for single-node clusters. Refer to
  [Skip Node Draining](../clusters/edge/cluster-management/skip-draining.md) for guidance on configuring draining behavior.

> - Upgrading the Palette from 4.4.x to 4.5.15 and later will now automatically renew the Certificate Authority (CA) 

Reviewer 2: ```suggestion
- Upgrading the Palette agent from 4.4.x to 4.5.15 and later will now automatically renew the Certificate Authority (CA) 
```

>   certificate for Mutating Webhook Handler (MWH). This was an issue that affected 4.4.x and prior versions and was 

Reviewer 2: ```suggestion
  certificate for the stylus-webhook mutatingwebhookconfiguration. This was an issue that affected 4.4.x and prior versions and was 
```
